plugin/controller/lib/impl/mcp/MCPControllerRegister.ts

```
@@ -334,20 +376,44 @@ export class MCPControllerRegister implements ControllerRegister {
     ]);
   }
 
-  sseCtxStorageRun(ctx: Context, transport: SSEServerTransport | StreamableHTTPServerTransport) {
+  sseCtxStorageRun(ctx: Context, transport: SSEServerTransport) {
+    const self = this;
     const mw = this.app.middleware.teggCtxLifecycleMiddleware();
     const closeFunc = transport.onclose;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)